Добавил рацию для сталкеров + Переписал в Английском Сидормате "PDA" на "KPK" .

SanecMan opened the pull request:
Рация мультиволновая.
Покупается в сидормате за 2500 RU.
Продаётся за 1250 RU.
Вроде-бы нигде нету дропа (я ни разу на локалке дроп не получил).
Баг только один: Где частота была ИИ там теперь только розовый текст.

### Changelog
#Stalker
-Добавлена рация для переговоров в сидормат за 15000 By SanecMan.
-"PDA" в Английском сидормате теперь "KPK".

SanecMan committed:
Добавил рацию для сталкеров + Переписал в Английском "PDA" на "KPK" .

Рация мультиволновая.
Покупается в сидормате за 2500 RU.
Продаётся за 1250 RU.
Вроде-бы нигде нету дропа (я ни разу на локалке дроп не получил).
Баг только один: Где частота была ИИ там теперь только розовый текст.

SanecMan committed:
Попросили поменять цену рации - получите.

С 2500 RU на 15000 RU .

SanecMan committed:
Revert "Попросили поменять цену рации - получите."

This reverts commit 09d7f21936cfe0f5e53fa19a86eb75dbc9b2294c.

SanecMan committed:
С 2500 RU на 15000 RU

Valtosin commented on code/game/stalker/machinery/sidormat.dm:
```diff
@@ -175,7 +175,8 @@ var/list/global_sidormat_list = list(
 	"Misc" = list(
 		/////////////////////////////////	������	/////////////////////////////////////////////
-		new /datum/data/stalker_equipment/stalker_pda("PDA",					"���",													/obj/item/device/stalker_pda,								4000,	ROOKIE),
+		new /datum/data/stalker_equipment/stalker_pda("KPK",					"���",													/obj/item/device/stalker_pda,								4000,	ROOKIE),
```
Может всё таки PDA? Например, тут описано: https://stalker-ru.gamepedia.com/PDA

SanecMan replied:
А после покупке на инглише это `KPK`.
Решать тебе: Или меняем название в сидормате у Английского языка на `PDA` или на `KPK`?

Popug commented:
Объясните такую цену за рацию. В моём понимании (я бомж, который разом накопил максимум лишь 12 штук) 15к это дохуя.

SanecMan commented:
> Объясните такую цену за рацию. В моём понимании (я бомж, который разом накопил максимум лишь 12 штук) 15к это дохуя.

Мне вчера пиздабол (ID его в дискорде 130631212690571264) написал что если рация будет стоить 2500 RU, то на ленту все хуй покладут. Собсна провёл голосование, и всех устроила цена в 15к. Собсна скрин: 
![image](https://user-images.githubusercontent.com/36476150/53935619-a1508380-40d9-11e9-8b9e-8f7543935550.png)
И ссылка на сообщение: https://discordapp.com/channels/469653484812238858/469655384370577448/552965324786630686

SanecMan committed:
Теперь KPK это PDA у английскоязычных

SanecMan committed:
В деревяный ящик добавил шанс выпадания рации.

Ящики такие деревянные разбивающиеся сами по себе редкость, но мне похуй. Шанс чуть более низок чем на 20.
